crypto/rsa/rsa_gen.c

```diff
@@ -47,7 +47,7 @@ static int rsa_builtin_keygen(RSA *rsa, int bits, BIGNUM *e_value,
      * When generating ridiculously small keys, we can get stuck
      * continually regenerating the same prime values.
```

Comment:
this comment should be updated then.
and some lines below there is a comment that mentions "bits >= 16" that one needs also updating.